src/exportcomponents/src/header/utils/generateUEID.ts

```diff
@@ -0,0 +1,8 @@
+export const generateUEID = () => {
+  let first: string | number = (Math.random() * 46656) | 0;
+  let second: string | number = (Math.random() * 46656) | 0;
```

Comment:
Can you enhance this to be a one liner instead of having 2 variables for more efficiency

Reply:
Refactored this helper